src/components/Counter/Counter.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import Counter from './Counter';
+
+describe('Counter', () => {
+  it('should render default value', async () => {
```

Comment:
It's actually not required to use `async` and `await` for such cases as it should work without them

Reply:
It's strange, but specifically for this component 'screen' function returns a Promise.